.gitignore

```diff
@@ -63,3 +63,4 @@ target/
 
 # PyCharm
 .idea
+deps/
```

Comment:
Deps is the folder with submodules so it should not be ignored

Reply:
But these should never be pushed to git right?

Reply:
If git says there are modifications, then you DONT have the version which this package specifies. Don't hide that information, it's important.

Reply:
I remembered that it did show already after a clean install. Therefore I added it to the gitignore. But now I did a clean recursive clone, I doesn't show. So I will revert.